hail/src/main/scala/is/hail/io/fs/AzureStorageFS.scala

```diff
@@ -305,8 +305,13 @@ class AzureStorageFS(val credentialsJSON: Option[String] = None) extends FS {
       override def close(): Unit = {
         if (!closed) {
           flush()
-          blobOutputStream.flush()
-          blobOutputStream.close()
+          try {
```

Comment:
This try/catch doesn't really help -- I think an error (`java.lang.IllegalStateException: Faulted stream due to underlying sink write failure`) is getting thrown during stream write and `blobOutputStream.flush()` is actually writing a corrupted file to Azure blob storage. :-(